contrib/completion/MAINTAINERS

```diff
@@ -1,2 +1,3 @@
 Tianon Gravi <admwiggin@gmail.com> (@tianon)
 Jessie Frazelle <jess@docker.com> (@jfrazelle)
+Vincent Bernat <vincent@bernat.im> (@vincentbernat)
```

Comment:
I think its great you want to be a maintainer so that we dont bother you on all completions if you cp the existing MAINTAINERS file to the zsh directory and add yourself there, we will ping you on the changes to zsh :D